query/physicalplan/aggregate.go

```diff
@@ -25,6 +25,7 @@ func Aggregate(
 	tracer trace.Tracer,
 	agg *logicalplan.Aggregation,
 	final bool,
+	only bool,
```

Comment:
I'm not 100% certain what this parameter does or why is it necessary. It might be cleaner to define an enum and merge final and only if necessary (since `only -> final`).

Reply:
I ran into some weird behavior when trying this out in prod anyways. Gonna backburner this PR for now